torch/distributed/distributed_c10d.py

```diff
@@ -513,6 +520,22 @@ def pg_coalesce_state(self) -> Dict[ProcessGroup, List[Union[_CollOp, P2POp]]]:
     def pg_default_device(self) -> Dict[ProcessGroup, torch.device]:
         return self._pg_default_device
 
+    @property
+    def pg_config_info(self) -> List[Dict[str, str]]:
```

Comment:
Where is this function used?

Reply:
This is currently used in one of the trace collection step to inject this information into the trace so the traces can be used for network replay and triaging.

Reply:
So this intends to be a user-facing API. Would you mind adding a short description under the def line? Thank you!

Reply:
I have added comments to clarify the usage of this function.